sea-orm-codegen/src/entity/base_entity.rs

```diff
@@ -125,6 +159,27 @@ impl Entity {
         }
     }
 
+    pub fn resolve_conjunct_relations_via_module_name(&self, name_resolver: &NameResolver) -> Vec<Ident> {
+        self.conjunct_relations
+            .iter()
+            .map(|con_rel| con_rel.resolve_via_module_name(name_resolver))
+            .collect()
+    }
+
+    pub fn resolve_conjunct_relations_to_module_name(&self, name_resolver: &NameResolver) -> Vec<Ident> {
+        self.conjunct_relations
+            .iter()
+            .map(|con_rel| con_rel.resolve_to_module_name(name_resolver))
+            .collect()
+    }
+
+    pub fn resolve_conjunct_relations_to_relation_name(&self, name_resolver: &NameResolver) -> Vec<Ident> {
+        self.conjunct_relations
+            .iter()
+            .map(|con_rel| con_rel.resolve_to_relation_name(name_resolver))
+            .collect()
+    }
+
```

Comment:
All these `resolve_*` methods (here, and in other files except those in `name_resolver.rs`) seems just a duplicate of the original methods few line below, e.g. `resolve_conjunct_relations_via_module_name` method is just `get_conjunct_relations_via_snake_case` method with name resolve applied. We should change the logic in the original method, i.e. applying name resolver in it.

Reply:
I didn't touch them originally, as that would have required more changes. Also, they are now more than just getters. Perhaps this one could be changed by later refactoring. 🤔